packages/datetime/src/common/classes.ts

```diff
@@ -27,29 +27,17 @@ export const DATE_INPUT_TIMEZONE_SELECT = `${NS}-date-input-timezone-select`;
 
 export const DATEPICKER = `${NS}-datepicker`;
 export const DATEPICKER_CAPTION = `${DATEPICKER}-caption`;
-export const DATEPICKER_CAPTION_CARET = `${DATEPICKER_CAPTION}-caret`;
 export const DATEPICKER_CAPTION_MEASURE = `${DATEPICKER_CAPTION}-measure`;
-export const DATEPICKER_CAPTION_SELECT = `${DATEPICKER_CAPTION}-select`;
 export const DATEPICKER_CONTENT = `${DATEPICKER}-content`;
-export const DATEPICKER_DAY = "DayPicker-Day";
-export const DATEPICKER_DAY_DISABLED = `${DATEPICKER_DAY}--disabled`;
-export const DATEPICKER_DAY_OUTSIDE = `${DATEPICKER_DAY}--outside`;
-export const DATEPICKER_DAY_SELECTED = `${DATEPICKER_DAY}--selected`;
-export const DATEPICKER_DAY_IS_TODAY = `${DATEPICKER_DAY}--isToday`;
-export const DATEPICKER_DAY_WRAPPER = `${DATEPICKER}-day-wrapper`;
 export const DATEPICKER_FOOTER = `${DATEPICKER}-footer`;
 export const DATEPICKER_MONTH_SELECT = `${DATEPICKER}-month-select`;
 export const DATEPICKER_YEAR_SELECT = `${DATEPICKER}-year-select`;
-export const DATEPICKER_NAVBAR = `${DATEPICKER}-navbar`;
-export const DATEPICKER_NAVBUTTON = `DayPicker-NavButton`;
 export const DATEPICKER_TIMEPICKER_WRAPPER = `${DATEPICKER}-timepicker-wrapper`;
 
 export const DATERANGEPICKER = `${NS}-daterangepicker`;
 export const DATERANGEPICKER_CALENDARS = `${DATERANGEPICKER}-calendars`;
 export const DATERANGEPICKER_CONTIGUOUS = `${DATERANGEPICKER}-contiguous`;
 export const DATERANGEPICKER_SINGLE_MONTH = `${DATERANGEPICKER}-single-month`;
-export const DATERANGEPICKER_DAY_SELECTED_RANGE = `${DATEPICKER_DAY}--selected-range`;
-export const DATERANGEPICKER_DAY_HOVERED_RANGE = `${DATEPICKER_DAY}--hovered-range`;
```

Comment:
Additionally, the following class names only existed in legacy datetime components and are thus no longer used:
```
DATEPICKER_DAY_WRAPPER
DATEPICKER_CAPTION_CARET
DATEPICKER_CAPTION_SELECT
DATEPICKER_NAVBAR
DATEPICKER_NAVBUTTON
DATERANGEPICKER_DAY_SELECTED_RANGE
DATERANGEPICKER_DAY_HOVERED_RANGE
```